src/commands/docker_buildx_image.yml

```diff
@@ -83,6 +87,7 @@ steps:
         fi
 
         noosinv docker.buildx "$@"
+      no_output_timeout: << parameters.no_output_timeout >>
```

Comment:
🐼 

For clarity, what about adding such a config close to the top of the code snippet (just under the `name` key)?
I fear this `timeout` setting may become difficult to spot or find if the command body grows in size.